service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/SampleRoboticsTestDataHelper.java

```diff
@@ -42,6 +42,27 @@ public static RoboticsData expectedRoboticsData(final String applicationType) {
             .build();
     }
 
+    public static RoboticsData expectedRoboticsData(final String applicationType, final String phoneNumber) {
```

Comment:
This introduces a lot of duplication with the method above, probably could invoke the method above and work with the partially prepared core data? What are your thoughts?